internal/commands/app/create.go

```diff
@@ -101,70 +104,66 @@ func (cmd *CommandCreate) Handler(profile *cli.Profile, ui terminal.UI, clients
 		return err
 	}
 
-	// TODO(REALMC-8135): Implement data-source flag for app create command
-	// dsCluster, dsClusterErr := cmd.inputs.resolveDataSource(clients.Realm, cmd.inputs.Project, app.ID())
-	// if dsClusterErr != nil {
-	// 	return dsClusterErr
-	// }
-	// service := realm.ServiceDescData{
-	// 	Config: map[string]interface{}{
-	// 		"clusterName":         dsCluster,
-	// 		"readPreference":      "primary",
-	// 		"wireProtocolEnabled": false,
-	// 	},
-	// 	Name: appName + "_cluster",
-	// 	Type: "mongodb-atlas",
-	// }
-	// Potentially try to use Import to create app Service
-	// _, dsErr := clients.Realm.CreateAppService(cmd.inputs.Project, app.ID(), service) // for output possibly
-	// if dsErr != nil {
-	// 	return dsErr
-	// }
-	// Won't need if import works
-	// zipPkgName, zipPkg, exportErr := clients.Realm.Export(
-	// 	cmd.inputs.Project,
-	// 	app.ID(),
-	// 	realm.ExportRequest{ConfigVersion: realm.DefaultAppConfigVersion},
-	// )
-	// if exportErr != nil {
-	// 	return exportErr
-	// }
-	// if idx := strings.LastIndex(zipPkgName, "_"); idx != -1 {
-	// 	zipPkgName = zipPkgName[:idx]
-	// }
-	// zipPkgPath := filepath.Join(dir, zipPkgName) // Is this the correct path
-	// if writeErr := local.WriteZip(zipPkgPath, zipPkg); writeErr != nil {
-	// 	return writeErr
-	// }
+	rowCount := 3
+	if cmd.inputs.DataSource != "" {
+		dsCluster, err := cmd.inputs.resolveDataSource(clients.Realm, groupID, newApp.ID)
```

Comment:
if we keep this function name, id rename this `var` to `dataSource`.  otherwise I'd consider renaming this to `resolveClusterName` to be more specific about what its doing (for if and when data lakes become a thing)